test/scripts/openshift-ci/infra/deploy.cert-manager.sh

```
@@ -17,33 +17,33 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "$SCRIPT_DIR/../common.sh"
 
 echo "⏳ Installing cert-manager"
-oc create namespace cert-manager-operator || true
+oc create namespace ${CERT_MANAGER_NAMESPACE} || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)